docs/fuel-101/sequencer.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Sequencer
+category: Fuel 101
+---
+
+# What is a Sequencer?
+
+> A sequencer is a component utilized in Layer 2 scaling solutions, such as Optimistic Rollups and certain types of Zero-Knowledge Rollups to order transactions.
+
+Since rollups strategically leverage the consensus and data availability of another layer 1 chain, they depend on a sequencer for transaction ordering.
```

Comment:
```suggestion
Since rollups strategically leverage the consensus and data availability of another Layer 1 chain, they depend on a sequencer for transaction ordering.
```